eng/Directory.Build.Data.props

```diff
@@ -120,6 +120,7 @@
     <!-- The set of tags that should be added to the packages -->
     <PackageCommonTags>windowsazureofficial;azureofficial</PackageCommonTags>
     <AzureCoreSharedSources>$(MSBuildThisFileDirectory)/../sdk/core/Azure.Core/src/Shared/</AzureCoreSharedSources>
+    <AzureCoreTestFramework>$(MSBuildThisFileDirectory)/../sdk/core/Azure.Core/tests/TestFramework/</AzureCoreTestFramework>
```

Comment:
Should this have the entire path to the csproj?

Reply:
it could.. I thought we intentionally just alias the path so that people can see the csproj name.